seed/serializers/properties.py

```diff
@@ -230,11 +230,11 @@ class PropertyStateWritableSerializer(serializers.ModelSerializer):
 
     # support naive datetime objects
     # support naive datetime objects
-    generation_date = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True)
-    recent_sale_date = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True)
-    release_date = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True)
-    analysis_start_time = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True)
-    analysis_end_time = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True)
+    generation_date = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True, required=False)
+    recent_sale_date = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True, required=False)
+    release_date = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True, required=False)
+    analysis_start_time = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True, required=False)
+    analysis_end_time = serializers.DateTimeField('%Y-%m-%dT%H:%M:%S', allow_null=True, required=False)
```

Comment:
The only set of endpoints that I see using this for write purposes are `/api/v2/property_views/` - which seems to be doing so indirectly via `PropertyViewAsStateSerializer`. Are you still using those property_views endpoints? They were slated to be deprecated, but we can add them back in.

If you aren't using those, just for my own curiosity, which endpoint(s) are directly or indirectly using `PropertyStateWritableSerializer`?

Reply:
It looks like the api/v2/property_view were not among the ones you asked me about in the other thread so I wasn't aware you were planning to deprecate those, soI may still need them.  

As far as I can tell, PropertyStateWriteableSerializer is only being used by PropertyViewAsStateSerializer which is being used by v2 PropertyViewViewSet and v2_1 PropertyViewSetV21.  Paul Munday and I set up the v2 PropertyViewViewSet.  

I have not had a chance to review v2.1  PropertyViewSetV21 so it's possible I don't need the v2 property_view endpoint if v2.1 properties endpoint is being ported into 3 and has all the same functionality

Reply:
> It looks like the api/v2/property_view were not among the ones you asked me about in the other thread so I wasn't aware you were planning to deprecate those, soI may still need them.

Sorry about that, I think we might have misunderstood which those endpoints actually were when we initially scoped out endpoints to deprecate, so I never included it in the list of endpoints to ask you about.

> As far as I can tell, PropertyStateWriteableSerializer is only being used by PropertyViewAsStateSerializer which is being used by v2 PropertyViewViewSet and v2_1 PropertyViewSetV21. Paul Munday and I set up the v2 PropertyViewViewSet.
> 
> I have not had a chance to review v2.1 PropertyViewSetV21 so it's possible I don't need the v2 property_view endpoint if v2.1 properties endpoint is being ported into 3 and has all the same functionality

Nice catch! I did look over `PropertyViewSetV21` and it doesn't provide the same functionality that `PropertyViewViewSet` does. We'll add `PropertyViewViewSet` back in! 


------------------------------
Finally, unless I hear otherwise, I'll go ahead and open another PR with these changes cherry-picked on top of latest develop. I'll close this PR once I do that.